packages/v4/src/content/design-guidelines/components/helper-text/helper-text.md

```diff
@@ -0,0 +1,46 @@
+---
+id: Helper text 
+section: components
+---
+
+Helper text is an on-screen field guideline that helps provide context regarding field inputs. 
+<br/>
+## Usage:
+**Helper text** is most commonly used in conjuction with form elements to provide additional hints and information for the user. It can also be used to explain why certain information is being requested, how to use a specific input or to suggest helpful hints. Helper text should always be visible and appears as a subheading, unlike [popover](https://www.patternfly.org/v4/components/popover/design-guidelines/) which display content in a new window.
+<br/>
+### When to use helper text:
+* Use helper text to provide imperative information regarding an input, such as character limits, password expectations, etc.  
+* Use colored helper text to convey the status of a user's input. For example, if a user has met the character requirements, display helper text in green font to show success. Similarly, if a user has not met a password requirement, display the helper text in red to show failure/error. <br/>
+Example: 
```

Comment:
You need a line underneath here (between "Example" and the image tag) cause right now it's showing up like this: 
<img width="626" alt="image" src="https://user-images.githubusercontent.com/51165119/173139698-207b75b0-8bbf-4b0b-bf6d-6deab9f696cd.png">

Also maybe bold "Example" so it stands out more?